src/server/api/enums/ClipboardEvents.ts

```diff
@@ -0,0 +1,4 @@
+enum ClipboardEvents {
```

Comment:
```suggestion
enum ClipboardEvent {
```

Reply:
@simonhamp seems like I cannot do that, what's your opinion on ClipboardActionEvent?

<img width="753" alt="image" src="https://github.com/user-attachments/assets/4159aa2e-a618-4142-bb98-915755f27cc7">

Reply:
If there's already a `ClipboardEvent` enum, can we just use that?

Reply:
I don't think it's an enum.

<img width="1275" alt="image" src="https://github.com/user-attachments/assets/4326c3fd-099b-46b7-a33a-da57b28bfad0">

Reply:
Ah ok, let's stick with `ClipboardEvents` - sorry for the turnaround